presto-lark-sheets/src/main/java/com/facebook/presto/lark/sheets/SheetsConnector.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.lark.sheets;
+
+import com.facebook.airlift.bootstrap.LifeCycleManager;
+import com.facebook.presto.lark.sheets.api.CachingSheetsApi;
+import com.facebook.presto.lark.sheets.api.SheetsApi;
+import com.facebook.presto.lark.sheets.api.SheetsSchemaStore;
+import com.facebook.presto.spi.connector.Connector;
+import com.facebook.presto.spi.connector.ConnectorMetadata;
+import com.facebook.presto.spi.connector.ConnectorRecordSetProvider;
+import com.facebook.presto.spi.connector.ConnectorSplitManager;
+import com.facebook.presto.spi.connector.ConnectorTransactionHandle;
+import com.facebook.presto.spi.session.PropertyMetadata;
+import com.facebook.presto.spi.transaction.IsolationLevel;
+
+import javax.inject.Inject;
+
+import java.util.List;
+import java.util.function.Supplier;
+
+import static java.util.Objects.requireNonNull;
+
+public class SheetsConnector
```

Comment:
To be more consistent, can we use LarkSheetsConnector as the class name?  Similarly, other class names should also use LarkSheets